src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ImportProcess.java

```diff
@@ -280,8 +282,8 @@ else if (coverage != null) {
         else if (coverage != null)
         {
         	try {
-        		final File directory = catalog.getResourceLoader().findOrCreateDirectory("data", workspace, store);
-        		final File file = File.createTempFile(store, ".tif", directory);
+        		final Resource directory = catalog.getResourceLoader().get(Paths.path("data", workspace, store));
+        		final File file = File.createTempFile(store, ".tif", directory.dir());
```

Comment:
On 01/12/15 23:55, NielsCharlier wrote:
> No problem, but mind that this method is huge and it would make it impossible to see what I actually changed.

Niels, I trust your discretion. For smaller methods I reformat where it 
does not make changes too hard to see. Note that you can add ?w=1 to a 
GitHub URL to ignore whitespace changes:
https://github.com/blog/967-github-secrets

-- 
Ben Caradoc-Davies <ben@transient.nz>
Director
Transient Software Limited <http://transient.nz/>
New Zealand